ClassRater/settings.py

```diff
@@ -119,3 +119,6 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+MEDIA_URL = '/media/'
+
+MEDIA_ROOT = 'homepage/media'
```

Comment:
Is this is the right way to set the **MEDIA_ROOT**, because it's hard-coded I wasn't sure? Is there a way to make it more general?

Reply:
It looks ok to me, but how does the url look when checking it out through `127.0.0.1:8000`?

Reply:
To be honest, I'm not sure. I didn't change the urls.py file or tried to access it from the frontend.
I only uploaded an image via Django admin UI so far.

Reply:
I think you should try to change the urls.py and try to access it, according to [this](https://stackoverflow.com/questions/5517950/django-media-url-and-media-root), so we know the setting is right

Reply:
nvm, I see your point from [here](https://docs.djangoproject.com/en/3.2/howto/static-files/#serving-files-uploaded-by-a-user-during-development)